scripts/integration.sh

```
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euo pipefail
+
+if [ -f .test.env ]; then
+    export $(grep -v '^#' .test.env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)